user_guide.adoc

```diff
@@ -1434,7 +1434,7 @@ complicated these `pos` expression would get without substreams.
 === Absolute positioning
 
 If you'd ever need to "escape" the limitations of a substream when
-using a `pos` key of a parse instance and address something absolutely
+using a `pos` key of a parsed instance and address something absolutely
```

Comment:
Technically, there are 2 kinds of instances: "parse instances" (which get its results from parsing a stream) and "value instances" (which get their result virtually, by evaluating an expression). Only "parse instances" have `pos` key (as they are indeed positioned somewhere in the stream). It's not a very pretty name, but technically it's there...

Reply:
okay. Then you are right.